programs/sponsors/sponsors.md

```diff
@@ -0,0 +1,61 @@
+# Sponsors
+
+JSON Schema uses [OpenCollective](https://opencollective.com/json-schema) and [GitHub Sponsors](https://github.com/sponsors) to gather money. This money is used to cover project expenses in a transparent way ([See the OpenCollective documentation](https://opencollective.com/how-it-works)), and your donation will help to sustain and grow the Project.
+
+Please go to our [OpenCollective page](https://opencollective.com/json-schema) or [GitHub Sponsors page](https://github.com/sponsors/json-schema-org) to become a backer or a sponsor.
+
+## Benefits of being a backer
+
+Backers donate $5 per month to the project, and get the following benefits:
```

Comment:
"Some is better than none" aside, $5 is tiny for a company, maybe I'd consider calling this "individual backer" to make it clear this tier is meant for individuals rather than companies?

Reply:
I'll use the term Individual Backer.